docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -267,87 +310,84 @@ Congratulate and counsel the mother on appropriate IYCF practices.
 
   Treatment coverage = :math:`\frac{\text{Children with MAM/SAM recieving treatment}}{\text{Total number of MAM/SAM kids}}`
 
 Treatment coverage should not be confused with geographical coverage. 
 
-  Geogpraphical coverage = :math:`\frac{\text{Healthcare facilities/communities delivering MAM/SAM treatments}}{\text{Total number of facilities/communities}}`
+  Geogpraphical coverage (GC) = :math:`\frac{\text{Healthcare facilities/communities delivering MAM/SAM treatments}}{\text{Total number of facilities/communities}}`
 
 Geographic coverage attempts to measure the *availability* of services which does not equate with the *service access* and *uptake*. 
 
 **Effectively covered** is the product of the treatment coverage and the treatment efficacy (proportion of the treated who were cured or 'cure-rate').
 
-  - Effective coverage ('met-need') = :math:`\text{treatment coverage}\times\text{treatment efficacy}`
+  - Effective coverage (EC) = :math:`\text{treatment coverage}\times\text{treatment efficacy}`
 
   - Not effectively covered = 1 - effective coverage
 
-For example, a service with treatment coverage of 50%, and a cure-rate of 90% is meeting 50% x 90% = 45% of need (effective coverage of 45%). 
-
 .. image:: effective_coverage_figure.svg
 
-
-| SAM programme treatment coverage: 48.8% (37.4 - 60.4) (this is a point coverage; assumes programmes are not good at case finding) [Isanaka_2021]_
+| SAM programme treatment coverage: 48.8% (37.4-60.4) (this is a point coverage; assumes programmes are not good at case finding) [Isanaka_2021]_
 | MAM programme treatment coverage: same as SAM for now until this website is updated https://acutemalnutrition.org/en/countries 
 
 
-.. note::
-
-  1) Add treatment efficacy for SAM and MAM programmes 
-  2) Discuss how coverage is estimated (SQUEAC surveys). Discuss difference in point-coverage and period-coverage. 
-  3) Update coverage values to 'effective coverage' values
-
+.. todo::
 
+  Discuss how coverage is estimated (SQUEAC surveys). Discuss difference in point-coverage and period-coverage. 
+ 
 
 .. list-table:: Baseline coverage data
-  :widths: 10 10 5 15 20
+  :widths: 10 10 5 15 20 20
   :header-rows: 1
 
   * - Location
     - Subpopulation
     - Coverage
-    - Value
+    - Value (CxE)
```

Comment:
my interpretation of Rajan's request today is to have Value (C) and Value (E) as two separate columns.  You might as well have the references in separate columns as well.

Reply:
done!